[wapisai] iP

wapisai opened the pull request:
# Slay all day 

- slay
- slay
- slay

1. slay
2. slay
3. slay

```
public class Main {
    public static void main(String[] args) {
        Application.launch(MainApp.class, args);
    }
}
```

- [ ] slay
- [ ] slay
- [ ] slay

💅💅💅💅💅💅💅💅💅

> [!NOTE]
> slay all day

[google.com](url)

one upon a `main` slayslayslay
**Slay slay slay**
*slay all day*
~~slay slay slay all day~~

Commit:
Add Gradle support

Eclipse-Dominator committed:
Bump gradle and lib version

Commit:
docs/README.md: Tweak document template

Let's tweak the docs/README.md (which is used as the user guide)
to fit Duke better. Specifically,

1. mention product name in the title
2. mention adding a product screenshot and a product intro
3. tweak the flow to describe feature-by-feature

wapisai committed:
Add Rename, Greet, Exit

wapisai committed:
Add Level-0

wapisai committed:
Add Echo

wapisai committed:
Add Echo

wapisai committed:
Merge remote-tracking branch 'origin/master'

wapisai committed:
Add Add, List

wapisai committed:
Add Mark as Done and Undone

wapisai committed:
Add ToDos, Events, Deadlines

wapisai committed:
Fix mark and unmark output string

wapisai committed:
Test using the I/O redirection technique

wapisai committed:
Add handle unknown input error and empty description for todo error

wapisai committed:
Add delete

wapisai committed:
Add Save

wapisai committed:
Move constants outside Main

wapisai committed:
Make message methods

wapisai committed:
Fix alignment and add javadoc

wapisai committed:
Fix load and write methods

wapisai committed:
Fix load method

wapisai committed:
Add dates and times

wapisai committed:
Merge branch 'branch-Level-8'
Add Date and Time Level-8

wapisai committed:
Add Ui class

wapisai committed:
Add Storage class

wapisai committed:
Add Parser class

wapisai committed:
Add add and delete message to Ui class

wapisai committed:
Add TaskList class

wapisai committed:
Merge branch 'branch-Level-7'
A-MoreOOP

wapisai committed:
Add packages

wapisai committed:
I messed up

wapisai committed:
Add Gradle

wapisai committed:
Add Junit

wapisai committed:
Add find

wapisai committed:
Merge branch 'branch-Level-9'
Requirements

ChrysanthemumT commented on src/main/java/duke/Parser.java:
```diff
@@ -0,0 +1,72 @@
+        String[] inputParts = original.split("\\s+");
+
+        //handle mark || unmark
+        if (inputParts.length == 2 && (inputParts[0].equals("mark") || inputParts[0].equals("unmark"))) {
```
Maybe use a switch statement, instead of long if-else statements

ChrysanthemumT commented on src/main/java/duke/Duke.java:
```diff
@@ -0,0 +1,63 @@
+    /**
+     * Runs the chat bot.
+     */
+    public void run() {
```
LGTM!

wapisai committed:
Use Checkstyle

ChrysanthemumT commented on src/main/java/duke/tasks/Task.java:
```diff
@@ -0,0 +1,74 @@
+package duke.tasks;
```
LGTM

ChrysanthemumT reviewed (commented):
This is my first PR review for CS2103T, reviewing coding standard issues.

looks good, overall

wapisai committed:
Add JavaFX

wapisai committed:
Add Intro

wapisai committed:
Add full duke functionality

wapisai committed:
Add JavaFX
Merge branch 'branch-Level-10'

wapisai committed:
processLine method: use if-else statements

processline method has low readability

Change if-else statements to switch case statements

Using switch case statements makes code more readable

wapisai committed:
Add assert statements

wapisai committed:
loadLine method uses if-else statements

loadLine method has low readability

Change if-else statements in loadLine to switch case statements

Using switch case statements improve readability

wapisai committed:
Add refactoring, improve readbility

wapisai committed:
Merge pull request #1 from wapisai/branch-A-Assertions

Branch a assertions

wapisai committed:
Merge pull request #2 from wapisai/master

Merging

wapisai committed:
Change output strings

wapisai committed:
Add Javadoc

wapisai committed:
I need to
Merge branch 'branch-A-JavaDoc'

wapisai committed:
Improve code quality

wapisai committed:
Resolve conflict

wapisai committed:
Revert "Improve code quality"

This reverts commit 22b6a09eebd2e4332af354d281bfbf1e526b4809.

wapisai committed:
Improve Code Quality

wapisai committed:
Improve Code Quality

wapisai committed:
Add MassOps

wapisai committed:
Change name

wapisai committed:
Add images

wapisai committed:
Update README.md

wapisai committed:
Update README.md

wapisai committed:
Update README.md

wapisai committed:
Update README.md

wapisai committed:
Fix write to file bug

wapisai committed:
Fix exit bug

wapisai committed:
Fix exit timing bug

wapisai committed:
Merge remote-tracking branch 'origin/master'

wapisai committed:
Fix write file bug

wapisai committed:
Update README.md

wapisai committed:
Update README.md

wapisai committed:
Update README.md

wapisai committed:
Add tests

wapisai committed:
Merge remote-tracking branch 'origin/master'

wapisai committed:
Fix test bug
